example-projects/advanced/phantom/repl.js

```diff
@@ -1,10 +1,20 @@
-if (phantom.args.length != 1) {
-    console.log('Expected a target URL parameter.');
-    phantom.exit(1);
+console.log("Phantom version: " + phantom.version.major + "." + phantom.version.minor + "." +  phantom.version.patch);
```

Comment:
Sure will do, I found it useful for understanding the version, but you're the boss :smile: